app/backend/src/clj/teet/integration/vektorio/vektorio_client.clj

```diff
@@ -0,0 +1,91 @@
+(ns teet.integration.vektorio.vektorio-client
+  (:require [teet.environment :as environment]
+            [org.httpkit.client :as client]
```

Comment:
use a unambiguous alias like http-client or htclient?

Reply:
i made a pr to change these globally since it occurred all over the codebase, so don't need to fix here.